### PR TITLE
Documentation revisions for readme on 2.0 API repo

### PR DESCRIPTION
**Pull request by johnwight**
## Description

I have revised the readme documentation on the 2.0 repo. These are proposed changes. My current thinking is that it is easier to submit revisions through the pull request rather than individual tickets in JIRA. If this is not easier, please let me know.

## Motivation and Context

Improve documentation usability

## How Has This Been Tested?

Tested in Atom Preview

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
-[X} Documentation
## Checklist:

- [ X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ X] My code follows the code style of this project.
- [ X] My change requires a change to the documentation.
- [ X] I have updated the documentation accordingly.
- [ X] I have read the **CONTRIBUTING** document.
- [ X] I have added tests to cover my changes.
- [ X] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
+The [Reporting API Guide](reporting-guide.md) provides configuration guidance and best practices for the ```/reports``` endpoint.
+
+## <a name="segments">Segments API Guide</a>
+The [Segments API Guide](https://github.com/AdobeDocs/analytics-2.0-apis/blob/master/reporting-guide.md) provides configuration guidance and best practices for the ```/segments``` endpoint.
+
+
+## <a name="calcmet">Using Calculated Metrics</a>
+The following examples show how to retrieve calculated metrics information:
+
+## <a name="virtual">Using Calculated Metrics</a>
```
Comment (mike2184) [inline README.md]: Using Calculated Metrics is listed twice here

File: README.md [hunk 2]
```diff
@@ -1,21 +1,60 @@
...
+
+## <a name="reporting">Reporting API Guide</a>
+The [Reporting API Guide](reporting-guide.md) provides configuration guidance and best practices for the ```/reports``` endpoint.
+
+## <a name="segments">Segments API Guide</a>
+The [Segments API Guide](https://github.com/AdobeDocs/analytics-2.0-apis/blob/master/reporting-guide.md) provides configuration guidance and best practices for the ```/segments``` endpoint.
+
+
+## <a name="calcmet">Using Calculated Metrics</a>
+The following examples show how to retrieve calculated metrics information:
```
Comment (mike2184) [inline README.md]: No link to calc metric examples

File: README.md [hunk 3]
```diff
@@ -1,21 +1,60 @@
...
+
+## <a name="virtual">Using Calculated Metrics</a>
+The following example shows an example request body to create a virtual report suite:
```
Comment (mike2184) [inline README.md]: Need link to virtual report suite examples here

File: README.md (hunk 2 above)
Comment (johnwight) [reply README.md]: I will remove this heading altogether until we can get some GET request samples instead of POST.

File: README.md (hunk 1 above)
Comment (johnwight) [reply README.md]: Removed.

**Pull request closed without merging**